SummaryOfPrimes.java

```diff
@@ -0,0 +1,23 @@
+public class SummaryOfPrimes {
+
+  public static void main(String[] args) {
+    System.out.println(primeSummary());
+  }
+
+  public static int primeSummary() {
+    int primeAmount = 0;
+  for (int i = 1; i < 2000000; i++) {
+    int factors = 0;
+      for (int e = 1; e <= i; e++) {
+        if (i % e == 0) {
+          factors++;
+        }
+      }
+      if (factors < 3) {
+        primeAmount += i;
+        System.out.println(i);
+      }
+    }
+    return primeAmount;
+  }
+}
```

Comment:
No not threads - I mean an alternative algorithm so maybe look at something like java implementations of 'the sieve of eratosthenes'. I'm assuming here there is an algorithm which will be more efficient and won't take too long to run but I could be wrong on that.